### DISCUSSION TITLE
Project Status?

### DISCUSSION THREAD

**Discussion by paul-snively** [epfldata/squid#68] [General]
1. Is this project still active? It's concerning how long it seems to have gone without activity.
2. If so, is there a point at which we can expect Scala 2.13 support?

Thanks!

**Comment by LPTK** [comment]
Hi Paul. This project is not really active, because it was mainly my PhD project and I have now moved on. Scala 3 introduces statically-typed quotation capabilities inspired by Squid, and that's where the future is IMHO (though Squid actually does quite a few things not supported in Scala 3 at the moment, such as custom IRs and static scope safety).

On the other hand, Squid still has a few academic users currently, so it is in minimal-maintenance mode. If there were demand for 2.13, I'd be happy to give it a couple of hours to update the project. I don't think it'd be very difficult.
